genai-perf/genai_perf/parser.py

```diff
@@ -106,6 +106,21 @@ class EndpointConfig:
 }
 
 
+def _check_payload_input_args(
+    parser: argparse.ArgumentParser, args: argparse.Namespace
+) -> argparse.Namespace:
+    """
+    Raise an error if concurrency or request-range is set
+    """
+
+    if args.prompt_source == ic.PromptSource.PAYLOAD:
+        if args.concurrency or args.request_rate:
+            raise ValueError(
```

Comment:
nit: Could you return this as a parser.error? That's more native to ArgParse and consistent with the other errors returned.